db/schema.rb

```diff
@@ -400,6 +400,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "gi_indicators", force: :cascade do |t|
+    t.string "name", default: "", null: false
+    t.date "year"
```

Comment:
It's very weird to be calling this attribute as `.year.year` in all the code. A year is a year, and a date, is a date. If you just want to store the year, I don't think `date` is the proper type here. I'd make it to be a integer.